flipperdevices/flipperzero-firmware
# [FL-1919] NFC rework with Flipper File Format

# What's new

- Change Nfc files structure
- Save and Load Nfc cards with flipper file
- Change Nfc EMV resources with flipper file format

# Verification 

- Verify all Nfc cards are saved and loaded correctly
- Verify bank card parsing AID, country and currency codes

# Checklist (do not modify)
